extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -107,6 +107,7 @@ define('ext.wikia.adEngine.adContext', [
 		context.slots.incontentPlayer = isProperCountry(instantGlobals.wgAdDriverIncontentPlayerSlotCountries) ||
 			isUrlParamSet('incontentplayer');
 
+		context.opts.scrollHandlerConfig = instantGlobals.wgAdDriverScrollHandlerConfig;
```

Comment:
I was thinking about how to combine `context.opts.scrollHandlerConfig` and `context.opts.enableScrollHandler` into one variable.
Initialise `context.opts.scrollHandlerConfig` only if scrollHandler is enabled and use the state of this object later.
However, not sure what it will improve/worsen, so let's skip my thoughts :smile: